docs/form.html

```diff
@@ -0,0 +1,445 @@
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='grid-alt' type='solid' color='#eeeded' ></box-icon>
+                            <span>Grids</span>
+                        </li>
+                       </a>
+                       <a href="../docs/image.html" class="text-deocration-none selected-text ">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='images' color='#eeeded' ></box-icon>
+                            <span>Images</span>
+                        </li>
+                       </a>
+                       <a href="../docs/list.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='list-ul' color='#eeeded' ></box-icon>
+                            <span>Lists</span>
+                        </li>
+                       </a>
+                       <a href="../docs/modal.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='card' color='#eeeded' ></box-icon>
+                            <span>Modal</span>
+                        </li>
+                       </a>
+                       <a href="../docs/navigation.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='navigation' color='#eeeded' ></box-icon>
+                            <span>Navbars</span>
+                        </li>
+                       </a>
+                       <a href="../docs/rating.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='star-half' type='solid' color='#eeeded' ></box-icon>
+                            <span>Ratings</span>
+                        </li>
+                       </a>
+                       <a href="../docs/toast.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='copy' color='#eeeded' ></box-icon>
+                            <span>Toasts</span>
+                        </li>
+                       </a>
+                       <a href="../docs/typography.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='font-family' color='#eeeded' ></box-icon>
+                            <span>Typography</span>
+                        </li>
```

Comment:
Change: Do not use inline styling.

Reply:
> Change: Do not use inline styling.

this is specific to the Icons which I am importing from (Boxicons).  so I believe it won't create any issue.